#NAV_MSGS
Header header
float32 WGS84_Lat
float32 WGS84_Lon
float32 WGS84_Alt
float32 NED_N
float32 NED_E
float32 NED_D
float32 Hori_Accuracy
float32 Vert_Accuracy
uint8 	FixType
uint8 	Flags
uint8 	Flags2
